packages/shared/lib/core/profile-manager/actions/subscribeToWalletApiEvents.ts

```diff
@@ -1,13 +1,15 @@
-import { get } from 'svelte/store'
-import { profileManager } from '../stores'
+import { get, Writable } from 'svelte/store'
+
 import {
     handleNewOutputEvent,
     handleSpentOutputEvent,
     handleTransactionInclusionEvent,
     handleTransactionProgressEvent,
-} from '../api/events'
+} from '../api'
+import { IProfileManager } from '../interfaces'
+import { profileManager as _profileManager } from '../stores'
 
-export function subscribeToWalletApiEvents(): void {
+export function subscribeToWalletApiEvents(profileManager: Writable<IProfileManager> = _profileManager): void {
```

Comment:
That's true, although in cases where you wanna set a default but the parameter is a union type then it would have to be specified. I know we could get rid of two imports by doing this in this case, but I just prefer to have it as explicit as possible.